synapse/models/orgs.py

```diff
@@ -542,9 +544,12 @@ def getModelDefs(self):
                     ('personnel:jobtitle', ('ou:jobtitle', {}), {
                         'doc': 'Title or designation for the contributed personnel.'}),
                 )),
+                ('ou:techname', {}, ()),
```

Comment:
Lets leave this off for now. I'd like to possibly create a unified entity/idea name form :)